docs/man_pages/project/creation/sample.md

```diff
@@ -3,13 +3,18 @@ sample
 
 Usage | Synopsis
 ------|-------
-General | `$ appbuilder sample [<Command>]`
+General | `$ appbuilder sample`    
+Additional | `$ appbuilder sample [<Command>]`
 
-Lists all available sample apps with name, description, GitHub repository and clone command.
+Lists all available sample apps with name, description, GitHub repository, and clone command. To clone a selected sample app,
+run its clone command as listed by `$ appbuilder sample`
 
 ### Attributes
-`<Command>` extends the `sample` command. You can set the following values for this attribute.
-* `clone` - Clones the selected sample app.
+`<Command>` is any related command that extends the sample command. You can run the following related commands:
```

Comment:
\`\<Command\>\` extends the \`sample\` command. You can set the following values for this attribute.